block-canvas/style.css

```diff
@@ -39,37 +38,14 @@ body {
 /*
  * Control the hover stylings of outline block style.
  * Unnecessary once block styles are configurable via theme.json
- * https://github.com/WordPress/gutenberg/issues/42794
+ * https://github.com/WordPress/gutenberg/issues/42794 and https://github.com/WordPress/gutenberg/issues/44417
  */
 .wp-block-button.is-style-outline>.wp-block-button__link:not(.has-background):hover {
 	background-color: var(--wp--preset--color--secondary);
 	color: var(--wp--preset--color--background);
 	border-color: var(--wp--preset--color--secondary);
 }
 
-/**
- * Currently table styles are only available with 'wp-block-styles' 
- * theme support (block css) thus the following needs to be included
- * since 'wp-block-styles' aren't used for this theme.
- * https://github.com/WordPress/gutenberg/issues/45065
- */
-.wp-block-table thead {
-	border-bottom: 3px solid;
-}
-.wp-block-table tfoot {
-	border-top: 3px solid;
-}
-.wp-block-table td,
-.wp-block-table th {
-	padding: var(--wp--preset--spacing--30);
-	border: 1px solid;
-	word-break: normal;
-}
-.wp-block-table figcaption {
-	font-size: var(--wp--preset--font-size--small);
-	text-align: center;
```

Comment:
It looks like these lines are still needed, but I recently opened a GB PR to address this: https://github.com/WordPress/gutenberg/pull/46172 (I meant to look at this a while ago and this reminded me!)

Reply:
I guess it depends on how opinionated Block Canvas should be. I quite like the default styles of this caption being a smaller font size and centered, but perhaps we should remove these styles so we're relying on the defaults Core provides?

We could also use the `caption` element in theme.json to at least apply the small font size, rather than using CSS here.